memov/core/manager.py

```diff
@@ -985,6 +993,87 @@ def _next_develop_branch(self, branches: dict[str, str]) -> str:
             i += 1
         return f"develop/{i}"
 
+    def list_branches(self) -> None:
+        """List all branches, marking current with *."""
+        branches = self._load_branches()
+        if not branches:
+            LOGGER.info("No branches found.")
+            return
+
+        current = branches.get("current")
+        for name, commit in branches["branches"].items():
+            prefix = "* " if name == current else "  "
+            LOGGER.info(f"{prefix}{name} -> {commit[:7]}")
+
+    def create_branch(self, name: str) -> None:
```

Comment:
```py
def list_branches(self) -> None:  # OK
def create_branch(self, name: str) -> None:  # Should return status
def delete_branch(self, name: str) -> None:  # Should return status
def switch_branch(self, name: str) -> None:  # Should return status
```

These methods return None but have multiple exit points (success/failure). Consider returning MemStatus for consistency with other methods like snapshot().